portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,77 @@
+import CommentHandler from '/CommentHandler.js';
+import UserHandler from '/UserHandler.js';
+
+export default class Controller {
+
+  constructor(document) {
+    this.username = undefined;
+    this.loggedIn = false;
+    this.document = document;
+  }
+
+  /** Shows the change name form*/
+  openNameForm() {
+    this.document.getElementById('name-form').classList.remove('hidden-elem');
+  }
+
+  /** Hides the change name form*/
+  closeNameForm() {
+    this.document.getElementById('name-form').classList.add('hidden-elem');
+  }
+
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   * @param {HTML template} commentCell The template for one comment
+   */
+  static async loadComments(commentBox, commentCell) {
+    try {
+      let commentArray = await CommentHandler.loadComments();
```

Comment:
You're accessing another static method here. Make this NOT a static method and actually instantiate a new instance of CommentHandler in the constructor (or `init_()` method, if you choose that instead). Then you can do something like:
```
this.commentHandler.loadComments();
this.commentHandler.commentsArray.map(....etc...
```

The point of this is that the CommentHandler instance holds everything about the comments, and the Controller doesn't. The Controller simply knows that it has a CommentHandler that will do all the work around comments.

If you wanted to get a little more in depth, you could pass the `document` into the CommentHandler when it is instantiated, so it also has access to modify the document as needed. Not sure if that 100% fits what you're doing here, but it is an acceptable pattern.